client/js/transports/openai-webrtc.mdx

```diff
@@ -20,16 +21,18 @@ const options: OpenAIServiceOptions = {
   }
 };
 
-const transport = new OpenAIRealTimeWebRTCTransport(options);
-let RTVIConfig: RTVIClientOptions = {
-  transport,
+let pcClient = new PipecatClient({
+  transport: OpenAIRealTimeWebRTCTransport.create(options),
   ...
-};
+});
+pcClient.connect();
 ```
 
-### Configuration Options
+## API Reference
+
+### Constructor Options
```

Comment:
Had an earlier comment about whether we're calling them constructor options or factory options